docs/B-Computations/logic.md

```diff
@@ -83,7 +87,7 @@ The C language supports three selection constructs:
 
 The flow charts for these three constructs are shown below:
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/selection.png)
+![selection](../../static/img/image42.png)
```

Comment:
Suggestion to provide meaningful alt text. Example: `Flow charts of the three selection constructs`